confignode/src/main/java/org/apache/iotdb/confignode/manager/node/NodeManager.java

```diff
@@ -650,6 +662,69 @@ public Map<Integer, BaseNodeCache> getNodeCacheMap() {
     return nodeCacheMap;
   }
 
+  /** Start unknownDataNodeDetector */
+  public void startUnknownDataNodeDetector() {
+    synchronized (scheduleMonitor) {
+      if (currentUnknownDataNodeDetectFuture == null) {
+        currentUnknownDataNodeDetectFuture =
+            ScheduledExecutorUtil.safelyScheduleWithFixedDelay(
+                unknownDataNodeDetectExecutor,
+                this::detectTask,
+                0,
+                UNKNOWN_DATANODE_DETECT_INTERVAL,
+                TimeUnit.MILLISECONDS);
+        LOGGER.info("Unknown-DataNode-Detector is started successfully.");
+      }
+    }
+  }
+
+  /**
+   * The detectTask executed periodically to find newest UnknownDataNodes
+   *
+   * <p>1.If one DataNode is continuing Unknown, we shouldn't always activate Transfer of this Node.
+   *
+   * <p>2.The selected DataNodes may not truly need to transfer, so you should ensure safety of the
+   * Data when implement transferMethod in Manager.
+   */
+  private void detectTask() {
+    List<TDataNodeLocation> newUnknownNodes = new ArrayList<>();
+
+    getRegisteredDataNodes()
+        .forEach(
+            DataNodeConfiguration -> {
+              TDataNodeLocation dataNodeLocation = DataNodeConfiguration.getLocation();
+              BaseNodeCache newestNodeInformation = nodeCacheMap.get(dataNodeLocation.dataNodeId);
+              if (newestNodeInformation != null) {
+                if (newestNodeInformation.getNodeStatus() == NodeStatus.Running) {
+                  oldUnknownNodes.remove(dataNodeLocation);
+                } else if (!oldUnknownNodes.contains(dataNodeLocation)
+                    && newestNodeInformation.getNodeStatus() == NodeStatus.Unknown) {
+                  newUnknownNodes.add(dataNodeLocation);
+                }
```

Comment:
Is it better to use NodeStatus.isNormalStatus() instead of only consider Running and Unknown status?